packages/client/src/runtime/core/types/Extensions.ts

```diff
@@ -60,11 +60,11 @@ export type DynamicQueryExtensionArgs<Q_, TypeMap extends TypeMapDef> = {
         : never
 }
 
-export type DynamicQueryExtensionCb<TypeMap extends TypeMapDef, _0 extends PropertyKey, _1 extends PropertyKey, _2 extends PropertyKey> =
+type DynamicQueryExtensionCb<TypeMap extends TypeMapDef, _0 extends PropertyKey, _1 extends PropertyKey, _2 extends PropertyKey> =
```

Comment:
@SevInf previously added these exports to fix an issue with the TypeScript compiler, see https://github.com/prisma/prisma/pull/19885

Are we sure exports are not needed anymore?

Reply:
There is a test that will fail if declaration for extended client is not created, so i think it's fine.